### PR TITLE
Update Contributing Guidelines

### LINKED ISSUE #199: Update Contributing Guidelines
Include the following in the CONTRIBUTING.md guidelines:
- [ ] A requirement to comment on the issue and wait for being assigned.

### PR DESCRIPTION
**Pull request by asmitdeb**
Updated CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Hi :wave: there, welcome to KWoC React project. We are glad that you considered
 ## Guidelines
 
 - We do not commit to `master` branch directly. We open pull requests and merge them in master after review.
+- Please comment under the issue and wait for it to be assigned to you.
```
Comment (harshkhandeparkar) [inline CONTRIBUTING.md]:
```suggestion
- Please comment under the issue you want to work on and wait for it to be assigned to you.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,6 +17,7 @@ Hi :wave: there, welcome to KWoC React project. We are glad that you considered
 - Each contribution begins with an issue in this repo.
   - Create the issue as necessary.
   - If you lack the technical knowhow, refer to [Learning Guide](learn.md) or ask for help to anyone.
+- Comment under the issue and wait for it to be assigned to you.
```
Comment (harshkhandeparkar) [inline CONTRIBUTING.md]:
```suggestion
- Comment under the issue you want to work on and wait for it to be assigned to you.
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (asmitdeb) [reply CONTRIBUTING.md]: Done, @harshkhandeparkar

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by harshkhandeparkar** [comment]
Thanks @asmitdeb!

**Pull request merged**